tools/release/image-binary-release/s3-release.sh

```diff
@@ -27,14 +27,14 @@ set -e
 ##########################################
 
 # check environment vars
-if [ -z ${s3_bucket_name} ]; then
-        s3_bucket_name="aws-otel-collector-test"	
+if [ -z "${s3_bucket_name}" ]; then
+  s3_bucket_name="aws-otel-collector-test"
 	echo "use default s3_bucket_name: ${s3_bucket_name}"
```

Comment:
Nitpick. Extra space need to be delete.

Reply:
I'm not sure I follow. What extra space are you talking about? Can you make a suggestion with the change?

Reply:
```suggestion
if [ -z "${s3_bucket_name}" ]; then
    s3_bucket_name="aws-otel-collector-test"
    echo "use default s3_bucket_name: ${s3_bucket_name}"
else
    echo "load s3_bucket_name from env var: ${s3_bucket_name}"
```